src/cbor.js

```diff
@@ -72,11 +70,15 @@ exports.unmarshal = (input, opts) => {
       if (val instanceof cbor.Tagged) {
```

Comment:
Ah, missed that, it is actually a very good point/guard. Can we add a comment so that the same question doesn't raise after?